build/lender/Lender.sol

```diff
@@ -41,9 +42,17 @@ contract Lender {
   /// @param f: the new value of the feenominator
   /// @return true if successful
   function setFee(uint256 f) external authorized(admin) returns (bool) {
+    require(f >= MIN_FEENOMINATOR, 'fee too high');
     feenominator = f;
     return true;
   }
+
+  /// @notice Disables fees
+  /// @return true if successful
+  function disableFees() external authorized(admin) returns (bool) {
```

Comment:
nah. this is not needed. 

make `setFee` able to take 0.

this obvs mean the `f >= MIN_FEENOMINATOR` would need to make an exception for 0 but thats far superior that writing another method to do the same thing.